breadcrumb.el

```diff
@@ -436,6 +436,7 @@ propertized crumbs."
       (bc--goto (selected-window) choice))))
 
 (provide 'breadcrumb)
+;;;###autoload (register-definition-prefixes "breadcrumb" '("breadcrumb-"))
 ;;; breadcrumb.el ends here
```

Comment:
As a side-note, if you place the local variables below it, then the statement in this line is no longer accurate. :grimacing:

Reply:
Feel free to fix.  Always found that "ends here" kinda silly anyway.  What's the point?

Reply:
> Feel free to fix. Always found that "ends here" kinda silly anyway. What's the point?

https://www.gnu.org/software/emacs/manual/html_node/elisp/Library-Headers.html says "Its purpose is to enable people to detect truncated versions of the file from the lack of a footer line."

Reply:
I've gone ahead and fixed it.

Reply:
> I've gone ahead and fixed it.

But not in this PR, please.  SOrry I should have been clearer.

Reply:
> "Its purpose is to enable people to detect truncated versions of the file from the lack of a footer line."

By the way, "enables people". Particular kinds of people who trade in truncated files coming up with genius ideas that no other language have.